mon-pix/app/router.js

```diff
@@ -65,4 +65,5 @@ Router.map(function() {
 
   // XXX: this route is used for any request that did not match any of the previous routes. SHOULD ALWAYS BE THE LAST ONE
   this.route('not-found', { path: '/*path' });
+  this.route('terms-of-service', { path: '/cgu' });
```

Comment:
Comme indiqué sur le commentaire, la dernière route doit-être la not-found

Reply:
Modifié